src/theme/DocSidebar/index.js

```diff
@@ -0,0 +1,260 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
I don`t know is it normal. This file auto-generated by docusaurus

Reply:
Yeah, it's ok. License headers & files should be preserved (doing otherwise is breaking the licensing rules).